File: app/src/main/java/me/eycia/LScrollView.java
package me.eycia;

import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;
import android.widget.ScrollView;

/**
 * Created by eycia on 16/5/30.
 */
public class LScrollView extends ScrollView {
    public LScrollView(Context context) {
        super(context);
    }

    public LScrollView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public LScrollView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public LScrollView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }

    @Override
    public boolean onTouchEvent(MotionEvent ev) {
        return false;
    }
}

File: app/src/main/java/me/eycia/LScrollView.kt
package me.eycia

import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import android.widget.ScrollView

/**
 * Created by eycia on 16/5/30.
 */
class LScrollView : ScrollView {
    constructor(context: Context) : super(context) {
    }

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
    }

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
    }

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int, defStyleRes: Int) : super(context, attrs, defStyleAttr, defStyleRes) {
    }

    override fun onTouchEvent(ev: MotionEvent): Boolean {
        return false
    }
}
